### ISSUE TITLE
[Provider Request]: Mercy Medical Center (Cedar Rapids, Iowa)

### ISSUE THREAD

**Issue by mhlas7** [fastenhealth/fasten-onprem#406]
### Provider/Institution Name

Mercy Medical Center (Cedar Rapids, Iowa)

### Provider/Institution Website

https://www.mercycare.org/

### Patient Portal Login Website

https://mychart.mercycare.org/mychart/Authentication/Login?

### Provider EHR Platform

Epic MyChart

### Anything else we should know?

I have seen and used this provider in previous versions of Fasten however I can not find it in the latest version.

**Comment by AnalogJ** [comment]
Hey @mhlas7 looks like the logo and website are incorrect, but the organization is still there.

**Comment by AnalogJ** [comment]
let me try to get that fixed for you.

**Comment by mhlas7** [comment]
Thank you @AnalogJ! Let me know when it is fixed and I will try again.

Just a thought, have you thought about allowing a provider search by the domain name? A lot of providers have very similar names so being able to search for the exact domain name would help narrow things down.

**Comment by AnalogJ** [comment]
yep, the issue is that a vast number of our providers are missing homepage urls and so it would be a pretty incomplete search experience:

https://toolbox.fastenhealth.com/catalog/editor

but yeah, I'd love to be able to search by NPI number, website, city/state, etc
I'm hoping as people start contributing changes to the catalog, we'll get to the point where it would make sense to implement that stuff.

**Comment by mhlas7** [comment]
That would be awesome!

I looked through the catalog and I am still not seeing "Mercy Medical Center". Let me know when it is updated so I can try again or if I need to try a different search term.

**Comment by AnalogJ** [comment]
I'll probably update it tomorrow, I'm working on a big change to add support for AdvancedMD EHR at the moment, and that'll add about 30k institutions at the same time. 

I'll cut a release of fasten-sources after that, and deploy that change, plus the Mercy Medical Center fix to the search index.

**Comment by mhlas7** [comment]
Awesome, thanks!

**Comment by AnalogJ** [comment]
Hey @mhlas7 can you confirm that this provider is fixed?

**Comment by mhlas7** [comment]
Hmm, I am not able to find the provider by searching "mercy medical center" or "mercy cedar rapids". Let me know if there is a better search term that makes it show up.

I am running version 1.1.1

**Comment by mhlas7** [comment]
Fixed in the latest version. Thanks @AnalogJ!!!

**Comment by AnalogJ** [comment]
Sorry it took so long! There's been a lot going on 😃

**Comment by mhlas7** [comment]
@AnalogJ sorry to reopen this but I am unable to find this provider again. I recently had to reset my fasten database and I have been re-adding all of my sources.

**Comment by AnalogJ** [comment]
Hey

Sorry, we're updating this repo to make it clear that Fasten Onprem is essentially unable to retrieve EHR records directly

see https://github.com/fastenhealth/fasten-onprem/issues/629

FollowMyHealth/Allscripts is supported by Fasten Connect, but we were not able to roll that functionality into Lighthouse
